README.md

```diff
@@ -502,9 +565,8 @@ Header parameters are:
 
 #### TABLE_NAME.data-revision
 
```

Comment:
Specifying column data types in expected result files was required in earlier versions of tempto, but it is now optional.